backend/api/quivr_api/modules/sync/utils/upload.py

```diff
@@ -92,14 +101,17 @@ async def upload_file_sync(
     knowledge_to_add = CreateKnowledgeProperties(
         brain_id=brain_id,
         file_name=upload_file.filename,
-        extension=os.path.splitext(
+        mime_type=os.path.splitext(
```

Comment:
We should use mimetype here. The upload_file from fastAPI does have a `content-type` which is a mimetype